auto/src/glew_tail.h

```diff
@@ -8,10 +8,10 @@
 #define GLEW_ERROR_GLX_VERSION_11_ONLY 3  /* Need at least GLX 1.2 */
 
 /* string codes */
-#define GLEW_VERSION 1
-#define GLEW_VERSION_MAJOR 2
-#define GLEW_VERSION_MINOR 3
-#define GLEW_VERSION_MICRO 4
+#define GLEW_VERSION GLEW_VERSION_MAJOR_STRINGGLEW_VERSION_MINOR_STRING
+#define GLEW_VERSION_MAJOR GLEW_VERSION_MAJOR_STRING
+#define GLEW_VERSION_MINOR GLEW_VERSION_MINOR_STRING
+#define GLEW_VERSION_MICRO GLEW_VERSION_MICRO_STRING
 
```

Comment:
I'm not in favour of these defines varying from one release to the next.  It's too fragile in terms of matching the header to a precompiled library.  I wonder if we could put the version information in a /* */ comment block in a manner that would be suitable for easy grepping, but would have no impact on the interface.